h2olog.cc

```diff
@@ -237,6 +244,8 @@ int main(int argc, char **argv)
 
     std::vector<std::string> cflags;
 
+    cflags.push_back(make_pid_cflag("TARGET_PID", h2o_pid));
```

Comment:
I suppose it is better to use more unique macro name(e.g. `H2O_PID`, `H2OLOG_H2O_PID` etc) for avoiding macro name conflict.

And this can be written as below

```c++
std::vector<std::string> cflags{make_pid_cflag("TARGET_PID", h2o_pid)};
```

Reply:
Makes sense! 27abea3afbb0e2b70bb81ba8f5e1cae114ef4ff7